Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -822,8 +825,27 @@ noncomputable instance Real.isROrC : IsROrC ℝ where
   norm_sq_eq_def_ax z := by simp only [sq, Real.norm_eq_abs, ← abs_mul, abs_mul_self z, add_zero,
     mul_zero, AddMonoidHom.zero_apply, AddMonoidHom.id_apply]
   mul_im_I_ax z := by simp only [MulZeroClass.mul_zero, AddMonoidHom.zero_apply]
+  le_iff_re_im := by simp only [AddMonoidHom.id_apply, AddMonoidHom.zero_apply, and_true,
+    forall_const]
 #align real.is_R_or_C Real.isROrC
 
+instance toStarOrderedRingK : StarOrderedRing K := by
```

Comment:
```suggestion
instance IsROrC.instStarOrderedRing : StarOrderedRing K := by
```

Reply:
```suggestion
instance IsROrC.toStarOrderedRingK : StarOrderedRing K := by
```
would be better I think; the `inst`naming convention doesn't apply for base-class instances (as those are usually auto-generated to start with `to`.

Reply:
Well, at the very least I think the `K` should be deleted.

Reply:
Whoops, I agree!